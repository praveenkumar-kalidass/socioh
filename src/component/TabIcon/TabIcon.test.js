import React from 'react';
import { render } from '@testing-library/react-native';

import TabIcon from './index';

describe('TabIcon', () => {
  it('should match snapshot', () => {
    const container = render(<TabIcon />);

    expect(container).toMatchSnapshot();
  });
});
