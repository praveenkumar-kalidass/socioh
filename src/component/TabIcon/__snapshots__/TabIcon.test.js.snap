// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabIcon should match snapshot 1`] = `
<Text
  allowFontScaling={false}
  style={
    Array [
      Object {
        "color": undefined,
        "fontSize": 12,
      },
      Array [
        Object {
          "color": "#000",
          "fontSize": 30,
        },
        Object {
          "color": "#000",
          "fontSize": 30,
        },
        Object {
          "color": "#b0e6e7",
          "fontSize": 24,
        },
      ],
      Object {
        "fontFamily": "Ionicons",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      Object {},
    ]
  }
>
  
</Text>
`;
